Jamulus.pro

```diff
@@ -1048,10 +1103,40 @@ contains(CONFIG, "opus_shared_lib") {
     DISTFILES += $$DISTFILES_OPUS
 }
 
+
+CONFIG(portaudio) {
+    DEFINES += USE_PORTAUDIO
+    HEADERS += src/portaudio-sound.h
+    SOURCES += src/portaudio-sound.cpp
+    mingw* {
```

Comment:
I have never used mingw compiler. I usually use the free Visual Studio compiler. Why do you use mingw? Any special reason?

Reply:
Mostly habit I guess. I didn't have Visual Studio installed.

Reply:
I would like to stick to Visual Studio for now if possible. Is there a chance that you install the free Visual Studio or is this no option for you?